client/interfaces.go

```diff
@@ -291,7 +291,7 @@ type InstanceServer interface {
 	RenameStoragePoolVolumeBackup(pool string, volName string, name string, backup api.StoragePoolVolumeBackupPost) (op Operation, err error)
 	DeleteStoragePoolVolumeBackup(pool string, volName string, name string) (op Operation, err error)
 	GetStoragePoolVolumeBackupFile(pool string, volName string, name string, req *BackupFileRequest) (resp *BackupFileResponse, err error)
-	CreateStoragePoolVolumeFromBackup(pool string, args StoragePoolVolumeBackupArgs) (op Operation, err error)
+	CreateStoragePoolVolumeFromBackup(args StoragePoolVolumeBackupArgs) (op Operation, err error)
```

Comment:
@stgraber I've changed this function's signature to bring it inline with `CreateInstanceFromBackup`. Although this is an API change, this function has not yet been published in any tagged release.

Reply:
I'm not sure that this is the right thing to do.

All other volume functions require a pool be explicitly passed, same is true for the API as you pointed out below where the pool name is encoded in the URL as the volumes collection sits under a pool.

So I think I'd actually expect the pool to be passed in this call.

That means we'll pretty much always disregard the pool name encoded in the backup but that's okay.

Reply:
@stgraber OK NP.

Reply:
@stgraber update